### PR TITLE
TLS deprecation banner

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

We are considering withdrawing support for TLS 1.0 and 1.1 as they are [old insecure protocols](https://www.comodo.com/e-commerce/ssl-certificates/tls-1-deprecation.php).

In order to ween the last few users off browsers that rely on old TLS, we shall show them a banner, briefly explaining the problem and recommending they upgrade their browser.

## What is the value of this and can you measure success?

Gives users of these browsers one last chance to upgrade their browsers before they no longer have access to our content.

We will use Ophan to track how many users see this banner, giving us a better understanding of how many people will be affected. This will inform whether further steps are needed, or whether we can just go ahead and deprecate.

## Does this affect other platforms - Amp, Apps, etc?

No

## Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

No

## Screenshots

screen shot 2018-05-29 at 11 34 30

screen shot 2018-05-23 at 14 09 19

## Tested in CODE?

- [ ] Test in CODE

### CODE REVIEW

File: static/src/stylesheets/module/_tls-warning.scss [hunk 1]
```diff
@@ -0,0 +1,83 @@
+$tls-warning-rule: darken($garnett-nav-background, 20%);
+
+.tls-warning {
+    background-color: $garnett-nav-background;
+    color: $neutral-1;
+    border-bottom: 1px solid $tls-warning-rule;
+
+    &__browser-links {
```
Comment (sndrs) [inline static/src/stylesheets/module/_tls-warning.scss]: using the `&` makes it very hard to grep for the class in future, we normally warn against this?

**Review by MatthewJWalls — APPROVED** [review]

**Comment by MatthewJWalls** [comment]
Scala looks good, did you manage to check it on code?

**Comment by SiAdcock** [comment]
@MatthewJWalls Yeah, the banner did not appear on the older browsers I was expecting it to. Not sure if CODE goes through `fastly-edge-cache`?

File: static/src/stylesheets/module/_tls-warning.scss (hunk 1 above)
Comment (SiAdcock) [reply static/src/stylesheets/module/_tls-warning.scss]: Good call, trying to be clever!

**Comment by MatthewJWalls** [comment]
@SiAdcock it's possible the header only applied it to theguardian.com I guess. Was the ab test switched on?

**Comment by SiAdcock** [comment]
It was! Haha, I'm not falling for that again!

**Review by sndrs — APPROVED** [review]

**Comment by mchv** [comment]
💎

**Pull request merged**